### PR TITLE
change response type from object to records

### PR DESCRIPTION
**Pull request by a-c-sreedhar-reddy**
With new version of bs records get compiles to objects.

### CODE REVIEW

**Review by MoOx — APPROVED** [review]

**Review by DCKT — APPROVED** [review]

**Comment by DCKT** [comment]
This is a breaking change feature, we should release a new npm version 

@MoOx Can you add this package on npm 😄 thanks !

File: src/ReactNativeImagePicker.re [hunk 1]
```diff
@@ -70,24 +70,24 @@ module ImagePicker = {
...
-    "fileName": option(string),
-    "path": option(string),
-    "latitude": option(float),
-    "longitude": option(float),
-    "timestamp": int,
-    "originalRotation": float,
+    didCancel: bool,
+    error: option(string),
+    customButton: string,
+    data: string,
```
Comment (phamhoaivu911) [inline src/ReactNativeImagePicker.re]: I think the type of `data` should be `option`. The [doc](https://github.com/react-native-community/react-native-image-picker/blob/master/docs/Reference.md#options) says we can pass `noData` option to disable generating data. Also, `data` is not generated for video.

Comment (MoOx) [reply src/ReactNativeImagePicker.re]: @a-c-sreedhar-reddy care to comment or update?

**Comment by a-c-sreedhar-reddy** [comment]
@MoOx @phamhoaivu911 made data optional.

**Review by phamhoaivu911 — APPROVED** [review]

**Review by MoOx — APPROVED** [review]
Thanks for this!

**Comment by MoOx** [comment]
How do you feel we release this as a patch release to keep the version number close to the original lib?
This is definitely a breaking change, but very easy to adjust right?

**Comment by phamhoaivu911** [comment]
It's ok for me 👌

**Comment by MoOx** [comment]
2.3.2 is out, sorry for the delay

**Pull request merged**